src/App.js

```diff
@@ -1,23 +1,30 @@
-import { useState, useEffect } from 'react'
-import { Switch, Route } from 'react-router-dom'
+import { useEffect, useState } from 'react'
+import { Route, Switch } from 'react-router-dom'
+import { cities } from './capstone.json'
+import HomePage from './components/HomePage/HomePage'
+import LikePage from './components/LikePage/LikePage'
 import Navigation from './components/Navigation/Navigation'
 import SearchPage from './components/SearchPage/SearchPage'
+import TripNavigation from './components/TripNavigation/TripNavigation'
 import TripPage from './components/TripPage/TripPage'
-import HomePage from './components/HomePage/HomePage'
 import YourTripsPage from './components/YourTripsPage/YourTripsPage'
-import TripNavigation from './components/TripNavigation/TripNavigation'
-import { cities } from './capstone.json'
 
 export default function App() {
   const [userInput, setUserInput] = useState('')
-  const [likedPlaces, setLikedPlaces] = useState([])
-  const [cards, setCards] = useState(loadFromLocal('cards') ?? [])
+  const [likedPlaces, setLikedPlaces] = useState(
```

Comment:
It might be a good time to create a custom hook `useLocalStorage` since you are using this in two places here already and the readability of your code would increase a lot!

Reply:
Great Idea! 👍🏻